person.rb

```diff
@@ -1,21 +1,23 @@
-class Person
-  attr_reader :id
-  attr_accessor :name, :age
+require './nameable'
 
-  def initilialize(age, name: 'Unknown', parent_permission: true)
-    @id = Random.rand(1..100)
+class Person < Nameable
+  attr_accessor :id, :name, :age
+
+  def initialize(id, age, name: 'Unknown', parent_permission: 'Unknown')
+    @id = id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update this `super` to this `super()` as the current one raises an error while creating a person